diff --git a/applications/DamApplication/custom_processes/dam_bofang_condition_temperature_process.hpp b/applications/DamApplication/custom_processes/dam_bofang_condition_temperature_process.hpp
index b3f9f4a9a3f..8717204ad6f 100644
--- a/applications/DamApplication/custom_processes/dam_bofang_condition_temperature_process.hpp
+++ b/applications/DamApplication/custom_processes/dam_bofang_condition_temperature_process.hpp
@@ -114,7 +114,7 @@ class DamBofangConditionTemperatureProcess : public Process
 
     //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
-    void ExecuteBeforeSolutionLoop() override
+    void ExecuteInitialize() override
     {
 
         KRATOS_TRY;
diff --git a/applications/DamApplication/python_scripts/dam_analysis.py b/applications/DamApplication/python_scripts/dam_analysis.py
index 864de02740e..bdc53917eb8 100644
--- a/applications/DamApplication/python_scripts/dam_analysis.py
+++ b/applications/DamApplication/python_scripts/dam_analysis.py
@@ -232,7 +232,7 @@ class DamAnalysis(AnalysisStage):
 
         # Initialize processes
         for self.process in self.list_of_processes:
-            self.process.ExecuteBeforeSolutionLoop()
+            self.process.ExecuteInitialize()
 
         for self.output_process in self.list_of_output_processes:
             self.output_process.ExecuteInitialize()
