diff --git a/applications/DamApplication/custom_processes/dam_bofang_condition_temperature_process.hpp b/applications/DamApplication/custom_processes/dam_bofang_condition_temperature_process.hpp
index b3f9f4a9a3f..8717204ad6f 100644
--- a/applications/DamApplication/custom_processes/dam_bofang_condition_temperature_process.hpp
+++ b/applications/DamApplication/custom_processes/dam_bofang_condition_temperature_process.hpp
@@ -114,7 +114,7 @@ class DamBofangConditionTemperatureProcess : public Process
 
     //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
-    void ExecuteBeforeSolutionLoop() override
+    void ExecuteInitialize() override
     {
 
         KRATOS_TRY;
